src/main/java/org/opentestsystem/delivery/logging/LoggingExceptionHandler.java

```diff
@@ -18,9 +22,23 @@
  * Called when unhandled exceptions in request mapping methods are thrown.
  */
 public class LoggingExceptionHandler {
+    private final static Logger LOG = LoggerFactory.getLogger(LoggingExceptionHandler.class);
+
     @ExceptionHandler
-    public static void handleException(HttpServletRequest request, Exception e) {
-        EventLogger logger = (EventLogger)request.getAttribute(LOGGER_NAME);
+    @ResponseStatus(INTERNAL_SERVER_ERROR)
+    public void handleExceptionFallback(final HttpServletRequest request, final Exception e) {
+        LOG.error("Problem handling request", e);
```

Comment:
This adds the error to the standard log path, even if an EventLogger isn't registered with the Request